src/drivers/Packet.py

```diff
@@ -0,0 +1,36 @@
+# Written by Walden Hillegass 
+# details of packet structure are in this document :
+# https://docs.google.com/document/d/e/2PACX-1vRbm1WKCdh5X4Ji4nGYI22LL2Y-oLjkN6hBZEl6U7xsOcApdzz1ynh-aBq82YZGA1eBMbsIvnchxTQM/pub
+
+import hashlib
+MAX_PAYLOAD = 500
+OVERHEAD = 12
+
+class Packet:
+    def __init__(self, packet_id : int, payload : bytes):
+        if (len(payload) > MAX_PAYLOAD):
+            raise PacketizerException("Payload is larger than max size. Use multi_packetizer() instead")
+        id_bytes = packet_id.to_bytes(4, 'little')
+        b = []
+        b.append(b'\x69\x69')
+        b.append(id_bytes)
+        b.append(payload)
```

Comment:
Payload can contain our EOT and EOF bytes (0x03, 0x04), Maybe a better solution is to put size at the beginning so we know when to stop?